src/App.jsx

```diff
@@ -4,15 +4,17 @@ import PureComponent from 'react-pure-render/component';
 import { Router, Route, IndexRoute, hashHistory } from 'react-router';
 
 import PageNotFound from './components/PageNotFound.jsx';
+import AppStructure from './components/AppStructure.jsx';
```

Comment:
Nit: Can you put this above the import for `PageNotFound.jsx`, since A comes before P, and according to our style guide, imports should be sorted alphabetically based on the module name (which, in this case, is `./components/AppStructure.jsx`).